### PR TITLE
Accessibility Improvements, Color Fixes, Menu Changes

### PR DESCRIPTION
**Pull request by kre8sions**
#### Screens

Screen Recording 2021-03-26 at 02 29 51 PM

Screen Recording 2021-03-26 at 02 32 06 PM

Screen Recording 2021-03-26 at 02 33 43 PM

Screen Recording 2021-03-26 at 02 37 05 PM

Screen Recording 2021-03-26 at 02 35 09 PM

Screen Recording 2021-03-26 at 02 35 57 PM

#### Breaking Changes
**NO**

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-574

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screenshot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut-off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_textarea/_textarea.scss [hunk 1]
```diff
@@ -44,14 +53,23 @@
...
     }
-    textarea:focus {
-      @include pb_textarea_focus_dark;
+    textarea:hover,
+    .pb_text_area_kit:hover {
+      background-color: rgba($white,.15);
     }
     &.error {
-      > textarea {
+      textarea {
```
Comment (coleerikson) [inline playbook/app/pb_kits/playbook/pb_textarea/_textarea.scss:72]: Suggestion: put this in the same level as `&.error`?

File: playbook/app/pb_kits/playbook/pb_textarea/_textarea.scss [hunk 2]
```diff
@@ -44,14 +53,23 @@
     textarea::placeholder {
       @include pb_body_light_dark;
     }
-    > textarea {
-      @include pb_textarea_dark;
+    textarea,
+    .pb_text_area_kit {
+      @include pb_body_dark;
+      background-color: rgba($white,$opacity_1);
+      border-color: rgba($white,.15);
```
Comment (coleerikson) [inline playbook/app/pb_kits/playbook/pb_textarea/_textarea.scss:60]: Suggestion: Space after comma, and add 0. ie `rgba($white, 0.15)`;

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 3]
```diff
@@ -113,6 +124,12 @@
         }
       } 
     }
+
+    button.trix-button.trix-button--icon.trix-button{
+      &:hover{
+        background-color: rgba($border_light,.50);
```
Comment (coleerikson) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:130]: Suggestion: Space after comma, and add 0. ie `rgba($white, 0.15)`;

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 4]
```diff
@@ -183,34 +201,62 @@
...
       border-top-left-radius: 0;
       border-top-right-radius: 0;
+      color: $text_dk_default;
     }
 
     trix-toolbar {
       .trix-button-row {
-        background: #2f3968;
-        border: 1px solid rgba(255, 255, 255, 0.1) !important;
+        background-color: rgba($white,.1);
```
Comment (coleerikson) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:217]: Suggestion: Space after comma, and add 0. ie `rgba($white, 0.15)`;

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 5]
```diff
@@ -183,34 +201,62 @@
...
+    
+    &:focus-within{
+      .trix-button-row, trix-editor {
+        background-color: rgba($white,.03);
+        border-color: $active_dark !important;
+      }
     }
 
     &.error {
       > trix-editor {
```
Comment (coleerikson) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:246]: Suggestion: put this in the same level as &.error?

**Review by coleerikson — APPROVED** [review]
Minor recommendations to my tastes but nothing major. Spacing primarily within `rgba` as well as adding a 0 to the decimals. Also reducing SCSS nesting if the "parent" only has 1 child.

**Comment by jasoncypret** [comment]
Looking pretty solid. I need to test light mode a bit more on Monday. FYI @kre8sions

File: playbook/app/pb_kits/playbook/pb_text_input/_text_input.scss [hunk 6]
```diff
@@ -21,15 +21,20 @@
...
+    }
+
     input:focus,
     .text_input:focus,
     input:-webkit-autofill:focus,
     .text_input:-webkit-autofill:focus {
       @include pb_textarea_focus;
+      border-color: $primary;
       -webkit-box-shadow: 0 0 0px 1000px $focus_input_light inset;
       transition: background-color 5000s ease-in-out 0s;
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_text_input/_text_input.scss:39]: Please add a transition to this border color (As well as all of them)

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 7]
```diff
@@ -64,6 +62,19 @@
...
+      border-color: $primary;
+    }
+  }
+  .trix-dialog.trix-dialog--link{
+      background-color: $white ;
+      border: 1px solid $border_light;
+      box-shadow: none;
+      margin-top: $space_sm;
+      margin-left: $space_sm;
+    }
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:77]: Add a transition. This is too abrupt

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 8]
```diff
@@ -183,34 +201,62 @@
 
     // Dark Mode Placeholder
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:202]: For some reason, the dark mode background transition is slower on the bar than it is in the text area. Let's sync those or perhaps remove the transition on the bar.

File: playbook/app/pb_kits/playbook/pb_select/_select.scss [hunk 9]
```diff
@@ -22,6 +24,9 @@
         background-color: $focus_input_light;
       }
     }
+    &:focus{
+      border-color: $primary;
+    }
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_select/_select.scss:29]: Transition

File: playbook/app/pb_kits/playbook/data/menu.yml [hunk 10]
```diff
@@ -21,12 +20,14 @@ kits:
...
   - flex
   - forms:
+    - checkbox
     - date_picker
     - file_upload
     - form
     - form_group
     - form_pill
     - radio
+    - rich_text_editor
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/data/menu.yml:30]: makes sense. 👍

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 11]
```diff
@@ -9,6 +9,7 @@
 @import "./trix_styles";
 
 
+
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:12]: Can we get rid of the extra space here?

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 12]
```diff
@@ -22,17 +23,19 @@
...
 
-
-
-  // trix-toolbar { display: none; }
   // Disables File Attatchment Button in Toolbar
   trix-toolbar [data-trix-button-group="file-tools"] {
     display: none;
   }
 
+  //Disables the predefined transition values
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:33]: Can we just get rid of this code too?

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 13]
```diff
@@ -22,17 +23,19 @@
...
+  // trix-toolbar, trix-editor {
+  //  transition: none !important;
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:35]: Can we just get rid of this code too?

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 14]
```diff
@@ -58,12 +61,28 @@
   &.focus-editor-targets {
     trix-toolbar {
       display: none;
+
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:64]: Can we get rid of this extra space?

**Review by jasperfurniss — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 15]
```diff
@@ -183,34 +200,63 @@
...
+      button.trix-button.trix-button--icon.trix-button{
+        &:hover{
+          background-color: rgba($white, 0.15);
+        }
+        &:focus{
+          background-color: $active_dark !important;
+        }
+      }
+      &:focus{
+        border-color:$active_dark;
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:233]: Space after selector

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 16]
```diff
@@ -183,34 +200,63 @@
...
+    &:focus-within{
+      .trix-button-row, trix-editor {
+        background-color: rgba($white, 0.03);
+        border-color: $active_dark !important;
+      }
     }
 
     &.error {
-      > trix-editor {
+      trix-editor {
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:246]: Space

Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:246]: Space

File: playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss [hunk 17]
```diff
@@ -183,34 +200,63 @@
...
+
+    .trix-dialog.trix-dialog--link.trix-active{
+      background-color: tint($bg_dark, 10%) !important;
+      border: 1px solid rgba($white, 0.1);
+      box-shadow: none;
+      margin-top: $space_sm;
+      margin-left: $space_sm;
+    }
+    .trix-button--dialog{
+      color: $white;
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss:259]: Space

File: playbook/app/pb_kits/playbook/pb_select/_select.scss [hunk 18]
```diff
@@ -61,9 +67,10 @@
   &.dark {
     select {
       @include pb_textarea_dark;
-      @include pb_title_4;
-      @include pb_title_dark;
-      background: $focus_input_dark;
+      @include pb_body_light_dark;
+      background: none;
+
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_select/_select.scss:72]: Any reason for this return?

File: playbook/app/pb_kits/playbook/pb_text_input/_text_input.scss [hunk 19]
```diff
@@ -46,18 +52,23 @@
...
-        @include pb_title_4;
-        @include pb_title_dark;
+        @include pb_body_dark;
         overflow: hidden;
+        background-color: rgba($white,$opacity_1);
+        border-color: rgba($white, 0.15);
+      }
+
+      input:hover,
+      .text_input:hover{
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_text_input/_text_input.scss:62]: Space

File: playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.scss [hunk 20]
```diff
@@ -97,8 +111,12 @@
       color: $white;
     }
     .typeahead-kit-select__option:hover {
-      background-color: tint($focus_input_dark, 5%);
+      background-color: $active_dark;
+    }
+    .typeahead-kit-select__indicator:hover{
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.scss:116]: Space

File: playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.scss [hunk 21]
```diff
@@ -4,9 +4,23 @@
...
+    cursor: pointer;
+  }
   .pb_typeahead_wrapper {
     position: relative;
 
+    .text_input_value_container{
+      cursor: text;
+    }
+    .text_input_indicators,
+    .pb_list_kit{
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.scss:17]: Space

File: playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.scss [hunk 22]
```diff
@@ -4,9 +4,23 @@
 @import "../tokens/shadows";
 
 [class^=pb_typeahead_kit] {
+  .typeahead-kit-select__option {
+    cursor: pointer;
+  }
   .pb_typeahead_wrapper {
     position: relative;
 
+    .text_input_value_container{
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.scss:13]: Space

File: playbook/app/pb_kits/playbook/pb_textarea/_textarea_mixin.scss [hunk 23]
```diff
@@ -68,3 +68,8 @@
 @mixin pb_textarea_focus_dark {
   @include pb_textarea_focus($focus_input_dark);
 }
+
+// Transitions ======================
+@mixin transition_default{
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_textarea/_textarea_mixin.scss:73]: Space

**Review by jasoncypret — CHANGES_REQUESTED** [review]
image
Rich text need some help

**Comment by jasoncypret** [comment]
This also appears off 
image

**Review by jasoncypret — APPROVED** [review]

**Pull request merged**